aspnetcore/mvc/controllers/actions.md

```diff
@@ -14,80 +14,86 @@ uid: mvc/controllers/actions
 
-Action methods should contain logic for mapping an incoming request to a business concern. Business concerns should typically be represented as services that your controller accesses through [dependency injection](dependency-injection.md). Actions then map the result of the business action to an application state.
-Actions can return anything, but frequently will return an instance of `IActionResult` (or `Task<IActionResult>` for async methods) that produces a response. The action method is responsible for choosing *what kind of response*; the action result *does the responding*.
+Actions can return anything, but frequently return an instance of `IActionResult` (or `Task<IActionResult>` for async methods) that produces a response. The action method is responsible for choosing *what kind of response*. The action result *does the responding*.
 
 ### Controller Helper Methods
 
-Although not required, most developers will want to have their controllers inherit from the base `Controller` class. Doing so provides controllers with access to many properties and helpful methods, including the following helper methods designed to assist in returning various responses:
+Controllers usually inherit from [Controller](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.controller), although this is not required. Deriving from `Controller` provides access to three categories of helper methods designed to return specific response types:
 
-**[View](../views/index.md)**
+#### 1. An empty response body
 
-Returns a view that uses a model to render HTML. Example: `return View(customer);`
+No `Content-Type` HTTP response header is included, since the response body lacks content to describe.
 
-**HTTP Status Code**
+There are two response types within this category: Redirect and HTTP Status Code. The Redirect response type differs from the HTTP Status Code type primarily in the addition of a `Location` HTTP response header.
 
-Return an HTTP status code. Example: `return BadRequest();`
+* **Redirect**
 
-**Formatted Response**
+    This type returns a redirect to an action or destination (using `Redirect`, `LocalRedirect`, `RedirectToAction`, or `RedirectToRoute`). For example, `return RedirectToAction("Complete", new {id = 123});` redirects to `Complete`, passing an anonymous object.
 
-Return `Json` or similar to format an object in a specific manner. Example: `return Json(customer);`
+* **HTTP Status Code**
 
-**Content negotiated response**
+    This type returns an HTTP status code. A couple helper methods of this type are `BadRequest` and `NotFound`. For example, `return BadRequest();` produces a 400 status code when executed.
```

Comment:
There are loads of methods in this category e.g. `Ok()`, `NotFound()`, and `Unauthorized()` -- when called without a `value`.

I was likely not specific enough in #1435. Many of these methods accept an optional `object value` parameter. The overloads that do this generally fall into the **content negotiated** category.